### ISSUE TITLE
Plot complex functions

### ISSUE THREAD

**Issue by constip** [mathics/Mathics#794]
When using Plot on a function with an imaginary part (e.g. `Plot[Ix,{x,1,10}]`, only an empty coordinate system is returned. WolframAlpha's solution is to show two grafs: Plot[{Re[Ix],Re[Ix]},{x,1,10}]`, which might be a better default...

**Comment by GarkGarcia** [comment]
Hey, I'm not sure I follow. Is the behaviour of `Plot` simply broken or are you suggesting we change `Plot`'s behaviour?

**Comment by cpiber** [comment]
I do not know what the intended output was, but an empty graph seems wrong to me.
Either an error or a proper graph needs to be returned (i.e. two graphs either separate or overlaid with two colours)

**Comment by constip** [comment]
I do not know what the intended output was, but an empty graph seems wrong to me.
Either an error or a proper graph needs to be returned (i.e. two graphs either separate or overlaid with two colours)

**Comment by wuyudi** [comment]
`Plot[Ix,{x,1,10}]` is different from `Plot[I x,{x,1,10}]`, `Ix` is a  var while `I x` means `I` times `x`

Even Mathematica return an empty img.

![image](https://user-images.githubusercontent.com/30024051/94446260-fb90c800-01da-11eb-9d57-87cb550d08e5.png)

**Comment by cpiber** [comment]
Indeed, however `Plot[I x,{x,1,10}]` returns an empty plot as well.

**Comment by rocky** [comment]
We are always looking for folks to help fixing if you want to get involved (and probably see this fixed sooner)

**Comment by GarkGarcia** [comment]
This probably an error in `Graphics` and isn't directly related to the web-interface.
